.ci/RELEASE_NOTES_GUIDE.md

```diff
@@ -1,37 +1,40 @@
-# Release Notes for Terraform/Magic Modules Auto-CHANGELOG
+# Terraform Release Notes Guide
 
-## Background
+Release notes from pull request messages in this repository are propagated
+into Terraform provider commits, attaching them to the feature added by
+an original MM PR. These are then collected as part of the release cut
+and shared direcltly with end users in the provider changelog & GitHub release
+details.
 
-The Magician bot has the ability to copy specifically formatted release notes
-from upstream Magic-Modules PRs to downstream PRs that have CHANGELOGS, namely
-PRs generated in the Terraform providers (GA, beta).
+Whomever merges a PR is responsible for the release note. For most contributors
```

Comment:
nit: I believe this should be `whoever` since it's the subject of the sentence.

```suggestion
Whoever merges a PR is responsible for the release note. For most contributors
```